index.html

```diff
@@ -2977,6 +2977,23 @@ <h1>DID Methods</h1>
             <a href="https://htmlpreview.github.io/?https://github.com/persistentsystems/corda-did-method/blob/master/corda_did_method.html">Corda DID method</a>
           </td>
         </tr>
+        <tr>
+          <td>
+            did:uns:
+          </td>
+          <td>
+            PROVISIONAL
+          </td>
+          <td>
+            <a href="https://docs.uns.network/">uns.network</a>
+          </td>
+          <td>
+            <a href="https://www.spacelephant.org/">Space Elephant SAS</a>
+          </td>
+          <td>
+            <a href="https://github.com/unik-name/did-method-spec/blob/main/did-uns/UNS-DID-Specification.md">UNS DID Method</a>
```

Comment:
FYI, publicKeyHex is not supported by DID Core.... without extension... recommend publicKeyBase58 or JWK.